editor/src/core/shared/project-file-types.ts

```diff
@@ -761,3 +765,14 @@ export type NodeModules = {
 
 // Key here is the full filename.
 export type ProjectContents = { [filepath: string]: ProjectFile }
+
+export function getTopLevelElementByExportsDetail(
+  file: ParseSuccess,
+  nameToLookFor: string,
+): UtopiaJSXComponent | null {
+  return (
+    file.topLevelElements.find(
+      (t): t is UtopiaJSXComponent => t.type === 'UTOPIA_JSX_COMPONENT' && t.name === nameToLookFor,
+    ) ?? null
+  )
+}
```

Comment:
Can you move this to `/editor/src/core/model/project-file-utils.ts` please, as we have a few functions close to this in there

Reply:
moved it in https://github.com/concrete-utopia/utopia/pull/4429/commits/71c5121c27040496610686db54e94c105d6c59cb